react/features/video-menu/components/web/ParticipantContextMenu.tsx

```diff
@@ -139,17 +159,17 @@ const ParticipantContextMenu = ({
     const _isAudioMuted = useSelector(state => isParticipantAudioMuted(participant, state));
     const _overflowDrawer = useSelector(showOverflowDrawer);
     const { remoteVideoMenu = {}, disableRemoteMute, startSilent }
-        = useSelector(state => state['features/base/config']);
+        = useSelector((state: IState) => state['features/base/config']);
     const { disableKick, disableGrantModerator, disablePrivateChat } = remoteVideoMenu;
-    const { participantsVolume } = useSelector(state => state['features/filmstrip']);
+    const { participantsVolume } = useSelector((state: any) => state['features/filmstrip']);
```

Comment:
nit : abit of inconsistency on the state's type - I see in places you declare it as `any`, some other as `IState`

Reply:
It's declared as `IState` for the features that have types (all the ones in /base). For the features that don't have types for their reducers it's `any`